week6.1-ex1.py

```diff
@@ -0,0 +1,51 @@
+from typing import List, Set, Iterable
+from time import time
+
+WORDS_FILE_PATH = r"C:\Users\97252\python\Notebooks\week06\resources\words.txt"
+word_to_search = "zwitterion"
+times_to_search = 1000
+
+
+def create_list_from_file() -> List:
+    """
+    The function open the file and create list of the words that it contains.
+    :return: The created list.
+    """
+    with open(WORDS_FILE_PATH, 'r') as words:
+        words_list = [line for line in words]
+    return words_list
+
+
+def create_set_from_file() -> Set:
+    """
+    The function open the file and create set of the words that it contains.
+    :return: The created set.
+    """
+    with open(WORDS_FILE_PATH, 'r') as words:
+        words_set = [line for line in words]
+    return set(words_set)
+
+
+def average_runtime(words: Iterable) -> float:
+    """
+    The function calculates the average time to search a word in the list/set received multiple times.
+    :param words: The list or the set to search in.
+    :return: The average time that take to search the current word several times.
+    """
+    total_time = 0.0
+    for _ in range(times_to_search):
```

Comment:
Please change the underscore index here.

Reply:
Why? I do not use the index itself, I just need it to execute the loop several times.
But I really changed in Pycharm and it does not comment to me that it is a variable that is not used, I would be happy to explain how it can be, I did not use it at all.

Reply:
Hey naama, you are right, you should use an _ here

Reply:
Ok:)